backend/siarnaq/gcloud/titan.py

```diff
@@ -49,6 +58,11 @@ def get_object(bucket: str, name: str, check_safety: bool) -> dict[str, str | bo
     blob = client.bucket(bucket).get_blob(name)
     match (check_safety, blob.metadata):
         case (False, _) | (True, {"Titan-Status": "Verified"}):
+            if get_raw:
+                return {
```

Comment:
for consistency, put a logging thing here?

Reply:
bump @j-mao

Reply:
this log seems unimportant